src/Idris/Doc/Keywords.idr

```diff
@@ -534,7 +552,8 @@ keywordsDoc =
   :: "else" ::= ifthenelse
   :: "forall" ::= forallquantifier
   :: "rewrite" ::= rewriteeq
-  :: "using" ::= ""
+  :: "using" ::= importDirectives
```

Comment:
`using` keyword is used also for named interface implementations that use another named implementation as a superimplementation. Maybe, if should be mentioned here also.

Reply:
I'm happy to merge a PR that someone would assemble to improve the `:doc` thing.
I originally left `using` blank because I did not know what to write. There are
least two other uses in the parser separate from the one added (and documented)
in this PR.